web/public/mgmt/config.yml

```diff
@@ -1233,19 +1233,21 @@ collections:
   - name: "funding-opportunities"
     label: "Fund Opps - Content"
     folder: "content/src/fundings"
-    summary: "/{{urlSlug}} - {{name}}"
+    summary: "[{{agency[0]}}] - {{name}}"
     identifier_field: "{{filename}}"
     slug: "{{filename}}"
     delete: true
+    sortable_fields: ["name", "agency"]
     create: true
     editor:
       preview: true
     fields:
       - { label: "Internal Notes", name: "notesMd", widget: "markdown", required: false }
       - { label: "File Name", name: "filename", widget: "write-once-read-only" }
       - { label: "Name", name: "name", widget: "string" }
+      - { label: "Display Name", name: "displayName", widget: "string" }
       - { label: "Url Slug", name: "urlSlug", widget: "string" }
-      - { label: "Id", name: "id", widget: "string" }
+      - { label: "Id", name: "id", widget: "write-once-read-only" }
       - {
           label: "Open Date",
```

Comment:
dust - I know we have Archieved fundings as well I assume this are being ignored / content doesn't care about them?

Reply:
I asked Content about the archived opps and they said not to include them.